### ISSUE TITLE
Generating Diverse High-Fidelity Images with VQ-VAE-2

### ISSUE THREAD

**Issue by patrick-g-zhang** [patrick-g-zhang/papers-i-read#11]
## Introduction and background
- compare pros and cons between gan-like generative model and likelihood generative model
- VQVAE
- PixelCNN
### related work
- The challenge of realistic music generation: modelling raw audio at scale. In VQVAE2 work, the top level and bottom level are complementary.
## Method

![img](https://pbs.twimg.com/media/Ef7YTwJU4AABeQ2?format=jpg&name=medium)
![img](https://pbs.twimg.com/media/Ef7a_QqU4AA-XrH?format=jpg&name=medium)

### Stage 1: Learning Hierarchy Latent Codes
- model local information separately from global information
- the prior model can be tailored to capture the specific correlations in that level
- the decoder is feed-forward not auto-regressive

### Stage 2 Learning Priors over Latent Codes
- learn a prior over the latent codes
- fitting prior distributions using neural networks from training data 
- each prior was trained seperately

### Trading off Diversity with Classifier Based Rejection Sampling
- likelihood based models need to fit all the modes present in the data
- propose an automated method for trading off diversity and quality

## Experiments and Evaluations
- Modeling High-Resolution Face Images
在如此高的分辨率的情况下，model 人脸确实是一个挑战，比如人脸的对称性
### Negative Log-Likelihood and Reconstruction Error
- likelihood reflect the overfitting of training and test set
### Precision - Recall Metric
recall and precision
